Sources/Commands/SwiftTool.swift

```diff
@@ -380,8 +380,8 @@ public class SwiftTool {
             (packageRoot ?? cwd).appending(component: ".build")
         
         // Setup the globals.
-        verbosity = Verbosity(rawValue: options.verbosity)
-        Process.verbose = verbosity != .concise
+        TSCUtility.verbosity = Verbosity(rawValue: options.verbosity)
+        Process.verbose = TSCUtility.verbosity != .concise
```

Comment:
feels like we should create a member variable here instead of setting TSCUtility.verbosity, no?

Reply:
or it this designed so that some downstream code that relies on the global continues to work?

Reply:
yes, I didn't replace who it works currently so I put the global value that is managed somewhere in the tool flow.